README.md

```diff
@@ -18,13 +18,16 @@
 
 - Navigate to our [community page](https://github.com/LearnWithCommunity)
 - Click on the highlighted area:
-  ![](https://user-images.githubusercontent.com/91655303/144694971-52b9ecc8-444e-4133-a74c-fc76d7b64692.png)
+
+![](https://user-images.githubusercontent.com/91655303/144694971-52b9ecc8-444e-4133-a74c-fc76d7b64692.png)
```

Comment:
Why remove the spaces?

Reply:
> Why remove the spaces?

With spaces.

![image](https://user-images.githubusercontent.com/51391473/144960066-ca039c14-c045-4e9b-add2-96e167fbb296.png)

Without spaces.

![image](https://user-images.githubusercontent.com/51391473/144960110-6b37c11e-1646-409a-9c7c-a9ea8ae13307.png)

I think it looks better without spaces. 🙂